src/test/kotlin/com/sourcegraph/cody/KeyboardShortcutTest.kt

```diff
@@ -28,11 +28,8 @@ class KeyboardShortcutTest : BasePlatformTestCase() {
             "cody.editCodeAction" to arrayOf("ctrl alt COMMA", "ctrl alt ENTER"),
             "cody.documentCodeAction" to arrayOf("ctrl alt H", "ctrl alt H"),
             "cody.testCodeAction" to arrayOf("ctrl alt G", "ctrl alt G"),
-
-            // Handles cody.inlineEditUndoAction, inlineEditCancelAction, & inlineEditDismissAction:
-            "cody.editCancelOrUndoAction" to arrayOf("ctrl alt BACK_SPACE", "ctrl alt BACK_SPACE"),
             "cody.editShowDiffAction" to arrayOf("ctrl alt D", "ctrl alt K"),
-            "cody.inlineEditAcceptAction" to arrayOf("ctrl shift EQUALS", "ctrl shift EQUALS"),
+            "cody.fixup.codelens.accept" to arrayOf("ctrl shift EQUALS", "ctrl shift EQUALS"),
```

Comment:
the action ids seem to be inconsistent now 😕

Reply:
I cannot help it, unless I want to change all of them.
I think we should try to make them the same as in cody where it makes sense :)
Here I need that for auto-binding without custom mapping one ID to the other (which I think is a way to go).